benefits/enrollment_switchio/views.py

```diff
@@ -0,0 +1,30 @@
+from django.http import JsonResponse
+from django.views.generic import TemplateView, View
+
+from benefits.core import models, session
+
+
+class IndexView(TemplateView):
+    """CBV for the enrollment landing page."""
```

Comment:
I don't think it's necessary for this docstring to be so specific to this view being a class-based view. We could just say "View handler" like the existing comments in `benefits.enrollment.views` or just "View".